deitel/chapter_3/ex_3_11/GradeBook.h

```diff
@@ -0,0 +1,17 @@
+#include <string>
+
+class GradeBook {
+public:
+    GradeBook(std::string);
```

Comment:
the parameters should have names, this will make the sources more self-documented